README.md

```diff
@@ -18,13 +18,13 @@ The LUKSO CLI is able to install multiple clients for running the node.
 
 ### Client versions
 
-| Client     | Version  | Release                                                      |
-| ---------- |----------|--------------------------------------------------------------|
-| Geth       | v1.13.5  | https://github.com/ethereum/go-ethereum/releases/tag/v1.13.5 |
-| Erigon     | v2.55.0  | https://github.com/ledgerwatch/erigon/releases/tag/v2.55.0   |
-| Prysm      | v4.0.8   | https://github.com/prysmaticlabs/prysm/releases/tag/v4.0.8   |
-| Lighthouse | v4.5.0   | https://github.com/sigp/lighthouse/releases/tag/v4.5.0       |
-| Teku       | v23.11.0 | https://github.com/Consensys/teku/releases/tag/23.11.0       |
+| Client     | Version  | Release                                                       |
+|------------|----------|---------------------------------------------------------------|
+| Geth       | v1.13.12 | https://github.com/ethereum/go-ethereum/releases/tag/v1.13.12 |
```

Comment:
I think that we should apply https://github.com/ethereum/go-ethereum/releases/tag/v1.13.13.

Please verify all others (if there are some major required updates, because of some bugs released).

Reply:
There are also other major updates that could be added here. Im worried that while testing some clients, any other that we support with the CLI can receive a new update, keeping the cycle of adding -> testing prolonged. I think we can make any new update in a new PR.

Reply:
Let's check next releases and confirm that there are no major issues fixed, e.g. Geth v1.13.13 fixes v1.13.12 bugs.